### PR TITLE
Add onecall API 3 support to allow access to daily summary

### PR DESCRIPTION
**Pull request by dbadia**
https://openweathermap.org/api/one-call-3

### CODE REVIEW

**Comment by Prominence** [comment]
Thanks for your contribution! I'll check the changes ASAP.

**Review by Prominence — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/prominence/openweathermap/api/OpenWeatherMapClient.java [hunk 1]
```diff
@@ -84,6 +84,18 @@ public OneCallWeatherRequester oneCall() {
         return new OneCallWeatherRequester(new RequestSettings(apiKey, timeoutSettings));
     }
 
+    /**
+     * One Call 3 API <a href="https://openweathermap.org/api/one-call-3">API</a>.
+     * Includes a weather summary statement in addition to the information provided by {@link #oneCall()}
+     * @return requester for retrieving one call weather information for the OneCall 3 API.
+     */
+    @SubscriptionAvailability(plans = ALL)
```
[the remark this answers is not in the document]
Comment (boblepepeur) [reply src/main/java/com/github/prominence/openweathermap/api/OpenWeatherMapClient.java:92]: +1 i need to use one call api 3.0, when will you merge ?

**Review by Prominence — APPROVED** [review]

File: src/main/java/com/github/prominence/openweathermap/api/OpenWeatherMapClient.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Prominence) [reply src/main/java/com/github/prominence/openweathermap/api/OpenWeatherMapClient.java]: > +1 i need to use one call api 3.0, when will you merge ?

Merged.

[the remark this answers is not in the document]
Comment (boblepepeur) [reply src/main/java/com/github/prominence/openweathermap/api/OpenWeatherMapClient.java]: You rocks !! Tested & Works like a charm thanksssss

**Pull request merged**